docs/specs/dynamic_model.md

```diff
@@ -2,12 +2,11 @@
 
 ## Introduction
 
-Simulation is a vital part of autonomous driving especially in Apollo where most of the testing happens via our simulation platform. In order to have a more accurate approach to our testing environment, Apollo 5.0 introduces Dynamic model which is Control-in-loop Simulation. It is possible to guage the how the ego car would perform while driving in autonomous mode using the Control-in-loop.
+Simulation is a vital part of autonomous driving especially in Apollo where most of the testing happens via our simulation platform. In order to have a more accurate approach to our testing environment, Apollo 5.0 introduces Dynamic model which is Control-in-loop Simulation. It is possible to gauge the how the ego car would actually perform while driving in autonomous mode, but virtually using the Control-in-loop simulation making it secure and efficient. It can also accelerate your development cycle as you can maximize your testing and enhance your algorithms even before entering the vehicle.
```

Comment:
Instead of saying 'Apollo 5.0 introduces Dynamic model which is Control-in-loop Simulation', better put it this way 'Apollo 5.0 introduces Dynamic model, which is used in Control-in-loop Simulation to represent real vehicles' dynamic characters'

Reply:
Agreed. Made the change